templates/cluster_resources.j2

```diff
@@ -127,8 +138,7 @@ primitive rsc_sap_{{ sid }}_ASCS{{ ascs_instance }} SAPInstance \
   params InstanceName={{ sid }}_ASCS{{ ascs_instance }}_{{ ascs_virtual_host }} \
      START_PROFILE="/sapmnt/{{ sid }}/profile/{{ sid }}_ASCS{{ ascs_instance }}_{{ ascs_virtual_host }}" \
      AUTOMATIC_RECOVER=false \
-  meta resource-stickiness=5000 failure-timeout=60 \
-     migration-threshold=1 priority=10
+  meta resource-stickiness=5000 {% if ensa_version == 1 %}failure-timeout=60 migration-threshold=1 priority=10{%- endif %}
```

Comment:
just wondering, if the `==` comparison works for jinja integer type

Reply:
Yes, it works. You can do tests in http://jinja.quantprogramming.com/ for example